packages/contracts/contracts/CollSurplusPool.sol

```diff
@@ -101,11 +102,4 @@ contract CollSurplusPool is CollSurplusPoolStorage, CheckContract, ICollSurplusP
             msg.sender == activePoolAddress,
             "CollSurplusPool: Caller is not Active Pool");
     }
-
-    // --- Fallback function ---
-
-    receive() external payable {
```

Comment:
Isn't it better to throw an error so the transaction gets `reverted` ?

Reply:
i think that the tx reverts too if there isn't a fallback function. No?

Reply:
:facepalm: you are right